source/dub/commandline.d

```diff
@@ -2396,7 +2398,10 @@ class DustmiteCommand : PackageBuildCommand {
 			if (m_combined) testcmd ~= " --combined";
 			// TODO: pass *all* original parameters
 			logDiagnostic("Running dustmite: %s", testcmd);
-			auto dmpid = spawnProcess(["dustmite", path.toNativeString(), testcmd.data]);
+
+			auto cmd = ["dustmite", path.toNativeString(), testcmd.data];
+			if (m_noRedirect) cmd ~= "--no-redirect";
```

Comment:
Please put the option before the test command (right after `"dustmite"` ideally). It's possible I may make future versions of DustMite pass through options after the test command to the test command, or otherwise interpret them differently.

Reply:
Good idea, fixed.